doc/help/elektra-spec.md

```diff
@@ -21,7 +21,7 @@ key database semantics (new plugins might be necessary, though).
 The most simple use is to enlist all keys that will be used by an application
 and maybe give a description for them (we use ini syntax in this document):
 
-```
+```ini
```

Comment:
I think you'd just write `apache` or `c++` and unknown formats should be ignored. Maybe it is easier to look at their language list on [github](https://github.com/isagalaev/highlight.js/tree/master/src/languages) or look at their CSS class list in the [docs](http://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases). There they even list some alias names.